swift/Samples/TicTacToe/Sources/Game/RunGameWorkflow.swift

```diff
@@ -89,17 +96,19 @@ extension RunGameWorkflow {
 
 extension RunGameWorkflow {
 
-    typealias Rendering = BackStackScreen
+    typealias Rendering = ModalContainerScreen // tuple of backstack and an optional modal
```

Comment:
This comment seems incorrect.

Reply:
Sorry, i forgot to remove it. This is a draft PR, I am still working through some cleanup.